src/main/java/com/board/config/boardconfiggui/data/outputmodels/pinconfig/PinConfig.java

```diff
@@ -122,24 +122,24 @@ public void removePinConfig(String portNumber, String pinNumber) {
     }
 
     /**
-     * Method to get all the bypass enabled pin configurations of the current port.
+     * Method to get all the bypass disabled pin configurations of the current port.
      *
-     * @return bypass supported pin configurations
+     * @return bypass disabled pin configurations
      */
-    public Map<String, List<String>> getBypassConfiguredPins() {
+    public Map<String, List<String>> getBypassDisabledPins() {
```

Comment:
You have changed the method name however didn't observe any changes related to the calling function in the commit. can you check once